eva/parser/utils.py

```diff
@@ -39,6 +40,22 @@ def parse_table_clause(expr: str):
     return stmt.from_table
 
 
+def parse_create_udf(udf_name: str, udf_file_path: str, type: str, **kwargs):
+    mock_query = f"CREATE UDF IF NOT EXISTS {udf_name}"
```

Comment:
We should make `IF NOT EXISTS` a function argument.

Reply:
in what situation would we not want IF NOT EXISTS when creating a UDF?

Reply:
If the `UDF` already exists, we don't want this to throw an error but instead raise a warning. 
This is important in exploratory analysis when a user creates the UDF once but runs the Python script multiple times.